### PR TITLE
fix: hide credential logs when disconnecting from DB server

### PR DESCRIPTION
**Pull request by hamsaaldrobi**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -45,7 +45,11 @@ module.exports = (options) => {
       return;
     }
 
-    logger.info(`Disconnecting from ${config.url}`);
+    if( config.showConnectionString ){
+      logger.info(`Disconnecting from ${config.url}`);
+    }else{
+      logger.info(`Disconnecting from server <hidden connection url>`);
+    }
```
Comment (kevinccbsg) [inline index.js]:
```suggestion
    if(config.showConnectionString){
      logger.info(`Disconnecting from ${config.url}`);
    } else {
      logger.info(`Disconnecting from server <hidden connection url>`);
    }
```

**Review by kevinccbsg — CHANGES_REQUESTED** [review]

**Review by kevinccbsg — APPROVED** [review]

**Review by UlisesGascon — APPROVED** [review]
LGTM! 🚀

File: index.js [hunk 2]
```diff
@@ -45,7 +45,11 @@ module.exports = (options) => {
       return;
     }
 
-    logger.info(`Disconnecting from ${config.url}`);
+    if(config.showConnectionString){
+      logger.info(`Disconnecting from ${config.url}`);
+    } else {
+      logger.info(`Disconnecting from server <hidden connection url>`);
+    }
```
Comment (Betisman) [inline index.js]:
```suggestion
    if (config.showConnectionString) {
      logger.info(`Disconnecting from ${config.url}`);
    } else {
      logger.info(`Disconnecting from server <hidden connection url>`);
    }
```

**Review by MatteoDiPaolo — APPROVED** [review]
LGTM

**Pull request merged**